tests/functional/services/api/imports/__init__.py

```diff
@@ -0,0 +1,102 @@
+import base64
+import json
+import os
+from typing import Dict, List, Optional, Union
+
+import requests
+
+from tests.functional.services.utils.http_utils import (
+    APIResponse,
+    InsufficientRequestDetailsError,
+    build_url,
+    get_api_conf,
+    get_headers,
+)
+
+
+def load_file(module_path: str, filename: str) -> str:
+    """
+    Given module path and filename, return file contents as string.
+    Assumes file is stored in folder with same name as module at same
+    path prefix.
+
+    :param module_path: path of invoking module
+    :type module_path: str
+    :param filename: name of file (including extension)
+    :type filename: str
+    :return: file contents
+    :rtype: str
+    """
+    directory, _ = os.path.splitext(module_path)
+    filename = os.path.join(directory, filename)
+    with open(filename) as f:
+        text = f.read()
+    return text
+
+
+def http_post_bytes(
```

Comment:
My only recommendation is move this to utils so it can be utilized by other tests

Reply:
done